app/views/templates/rulingCoverLetterTemplate.scala.html

```diff
@@ -136,21 +136,90 @@ <h3 class="govuk-heading-m">@messages("pdf.ruling.section.ifYouDisagree.title")<
                 <li>@messages("pdf.ruling.section.ifYouDisagree.2")</li>
             </ul>
             <p class="govuk-body-s">
-                @messages("pdf.ruling.section.ifYouDisagree.cannot")
+                @messages("pdf.ruling.section.ifYouDisagree.within")
             </p>
             <p class="govuk-body-s">
-                @messages("pdf.ruling.section.ifYouDisagree.moreInfo") <a class="govuk-link" href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/877291/HMRC1.pdf">@messages("pdf.ruling.section.ifYouDisagree.read")</a>
+                @messages("pdf.ruling.section.ifYouDisagree.cannot")
             </p>
             <h4 class="govuk-heading-s">@messages("pdf.ruling.section.askForReview.title")</h4>
             <p class="govuk-body-s">
-                @Html(messages("pdf.ruling.section.askForReview.email", messages("pdf.ruling.email")))
+                @messages("pdf.ruling.section.askForReview.reviewOfficer")
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.askForReview.ifYouDisagree")
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.askForReview.furtherInfo")
+            </p>
+            <h4 class="govuk-heading-s">@Html(messages("pdf.ruling.section.askForReview.guidance.url"))</h4>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.askForReview.requestReview")
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.askForReview.solsTeam")
+            </p>
+            <ul class="govuk-list govuk-list--bullet govuk-body-s">
+                <li>@messages("pdf.ruling.section.askForReview.solsTeam.1")</li>
+                <li>@messages("pdf.ruling.section.askForReview.solsTeam.2")</li>
+                <li>@messages("pdf.ruling.section.askForReview.solsTeam.3")</li>
+                <li>@messages("pdf.ruling.section.askForReview.solsTeam.4")</li>
+                <li>@messages("pdf.ruling.section.askForReview.solsTeam.5")</li>
+            </ul>
+            <p class="govuk-body-s">
+                @messages("pdf.ruling.section.askForReview.solsTeam.cannotWrite")
+            </p>
+            <p class="govuk-body-s">
+                @Html(messages("pdf.ruling.section.askForReview.solsTeam.emailText", messages("pdf.ruling.section.askForReview.solsTeam.email"), messages("pdf.ruling.section.askForReview.solsTeam.email")))
+            </p>
+            <p class="govuk-body-s">
+                @messages("pdf.application.footer.solicitors.office.and.legal.services.address")
```

Comment:
Currently the br tags are showing on the page and not rendering properly. To render the br tags properly, please update this line to `@Html(messages("pdf.application.footer.solicitors.office.and.legal.services.address"))` thanks